app/src/main/java/ch/sdp/vibester/api/InternetState.kt

```diff
@@ -0,0 +1,31 @@
+package ch.sdp.vibester.api
+
+import android.content.Context
+import android.net.ConnectivityManager
+
+class InternetState private constructor(){
+    companion object{
+        /**
+         * Get Internet Connexion Status
+         * @return return true if connected; false otherwise
+         */
+        fun getInternetStatus(ctx: Context): Boolean{
+            val connectivityManager = ctx.getSystemService(Context.CONNECTIVITY_SERVICE) as ConnectivityManager
+            val networkInfo = connectivityManager.activeNetworkInfo
+            var isNetwork:Boolean = false

```

Comment:
```suggestion
            var isNetwork: Boolean = false

```

Reply:
I found the name isNetwork a bit weird since you're not using it to see if the object is a network, consider renaming it.